### ISSUE TITLE
Add license to module

### ISSUE THREAD

**Issue by snadi** [SoftwareEngineeringEducationInitiative/CodeReviewModule#1]

**Comment by snadi** [comment]
@MichaelHilton @hwright @MichaelHilton what license do you suggest we use?

Should we use some attribution license? Or do you want it to be completely open?

**Comment by MichaelHilton** [comment]
I think this is a very important point. I can see benefits to both sides.  I would probably lean towards a CC remix-able attribution license.  What do you think we should do?
Thanks,
Michael

**Comment by snadi** [comment]
My personal opinion (similar to yours) is that all material should be open but there should be attribution in the form of putting a link to the repo. How they use the material and why is up to them. The attribution is nice because it will help us see this grow and also give credit to the people involved. 

I guess the question here is would we be upset if someone builds a book based on this material and makes money out of it? If the answer is yes, we need to take that into account.

**Comment by MichaelHilton** [comment]
I agree with your position. What do you think about a license such as:
CC attribution?
https://creativecommons.org/licenses/by/4.0/

**Comment by snadi** [comment]
Yup, I agree.
